ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.txt

```diff
@@ -44,9 +44,9 @@ hideall "--sync--"
 193.0 "Magitek Slug" sync /:Regula van Hydrus:10DB:/
 
 # Rotation resumes here whether or not turrets are successful
-199.3 "Bastardbluss" sync /:Regula van Hydrus:10DA:/ window 100,5
+199.3 "Bastardbluss" sync /:Regula van Hydrus:10DA:/ window 100,2
```

Comment:
I think this is fine but part of me says don't add syncs for things this close, just to avoid weird timline resyncing.  5s seems safe, but 3 feels iffy.

Reply:
I suppose you're right. Papering over old work for even older content will do that. I'll be very careful to watch for that if I ever do the ARR content, since it will probably come up there as well. Thanks for the heads up!